aws-rds-dbparametergroup/src/test/java/software/amazon/rds/dbparametergroup/AbstractTestBase.java

```diff
@@ -49,6 +49,32 @@ public class AbstractTestBase {
     protected static final Map<String, Object> PARAMS;
     protected static final Map<String, Object> RESET_PARAMS;
     protected static final RequestLogger EMPTY_REQUEST_LOGGER;
+    protected static final List<Parameter> MANY_CURRENT_PARAMETERS_SORTED;
+    protected static final List<Parameter> MANY_DEFAULT_PARAMETERS_SORTED;
+
+    private static List<Parameter> helpBuildManyParametersConstant(String parameterValue) {
+        String[] sortedParameterNames = new String[] {
+                "binlog_format", "character_set_server", "collation_server", "event_scheduler",
+                "innodb_autoinc_lock_mode", "innodb_buffer_pool_instances", "innodb_lock_wait_timeout",
+                "innodb_log_file_size", "innodb_print_all_deadlocks", "innodb_read_io_threads", "innodb_write_io_threads",
+                "lock_wait_timeout", "log_bin_trust_function_creators", "lower_case_table_names" ,"max_allowed_packet",
+                "net_write_timeout", "performance_schema", "query_cache_size", "query_cache_type", "read_only",
+                "skip_name_resolve", "slave_parallel_type", "slow_query_log", "sql_mode", "time_zone", "tx_isolation"
+        };
```

Comment:
Hardcoding these (assuming the exact parameter names provide no additional behavior-changing semantics) should be avoided. Let's generate a set of random 5-20 character random strings upon a test run.

The helper could shrink to:

```Java
private List<String> generateRandomStringList(int listLen, int wordLen, String alphabet) {
    Stream.generate(() -> common.test.randomString(wordLen, alphabet)).limit(listLen).collect(Collectors.toList())
}
```